src/openms/source/METADATA/ID/sources.cmake

```diff
@@ -0,0 +1,20 @@
+### the directory name
+set(directory source/METADATA/ID)
+
+### list all filenames of the directory here
+set(sources_list
+IdentificationData.cpp
+IdentificationDataConverter.cpp
```

Comment:
where are all the cpp files? are all the ID structures header-only?

Reply:
Most of them currently are.